.github/workflows/cvmfs-ci.yml

```diff
@@ -29,9 +29,9 @@ jobs:
             SCRAM_ARCH: "el8_amd64_gcc12"
             ROOT: "ROOT v6.32/06"
           - IMAGE: "cmscloud/al8-cms"
-            CMSSW_VERSION: "CMSSW_14_2_0_pre4_ROOT635" 
+            CMSSW_VERSION: "CMSSW_15_1_ROOT6_X_2025-03-30-2300"
```

Comment:
Why is a nightly build needed for this test? Nightly builds get cleaned every now and then

Reply:
That's true, but the problem with `CMSSW_14_2_0_pre4_ROOT635` is that it uses a ROOT nightly build from last November:
```bash
lxplus806:~/cmssw> cd CMSSW_14_2_0_pre4_ROOT635/src
lxplus806:~/cmssw/CMSSW_14_2_0_pre4_ROOT635/src> cmsenv
lxplus806:~/cmssw/CMSSW_14_2_0_pre4_ROOT635/src> root --version
ROOT Version: 6.35.01
Built for linuxx8664gcc on Nov 18 2024, 11:40:50
From heads/master@tags/v6-35-01
```

The ROOT interfaces are not stable during development phase, and the AD developments are working only for either the latest release (ROOT 6.34), or a more recent commit in ROOT `master`. So we need some way to test a current ROOT mater, and using a CMS ROOT6_X integration build is a temporary solution for that.